### DISCUSSION TITLE
Major library issue some of this library are missing in the requirements list all the libraries ASAP.

### DISCUSSION THREAD

**Discussion by Ayush-Sachan01** [HackByte3-0/SafeMind-AI#15] [Q&A]
list the libraries.

**Comment by aryanyk** [comment]
absl-py                      2.1.0      
annotated-types              0.7.0      
asgiref                      3.8.1      
astunparse                   1.6.3      
cachetools                   5.5.2      
certifi                      2024.8.30  
charset-normalizer           3.4.0      
colorama                     0.4.6      
contourpy                    1.3.0      
cycler                       0.12.1     
decorator                    4.4.2      
Django                       5.1.2      
django-allauth               65.0.2     
django-jsonfield             1.4.1      
django-widget-tweaks         1.5.0      
facenet-pytorch              2.6.0      
fer                          22.5.1     
ffmpeg                       1.4        
filelock                     3.16.1     
flatbuffers                  24.3.25    
fonttools                    4.54.1     
fsspec                       2024.9.0   
gast                         0.6.0      
google-ai-generativelanguage 0.6.15     
google-api-core              2.24.2     
google-api-python-client     2.166.0    
google-auth                  2.38.0     
google-auth-httplib2         0.2.0      
google-generativeai          0.8.4      
google-pasta                 0.2.0
googleapis-common-protos     1.69.2
grpcio                       1.71.0
grpcio-status                1.71.0
h5py                         3.12.1
httplib2                     0.22.0
idna                         3.10
imageio                      2.36.0
imageio-ffmpeg               0.5.1
Jinja2                       3.1.4
keras                        3.6.0
kiwisolver                   1.4.7
libclang                     18.1.1
Markdown                     3.7
markdown-it-py               3.0.0
MarkupSafe                   3.0.2
matplotlib                   3.9.2
mdurl                        0.1.2
ml-dtypes                    0.4.1
moviepy                      1.0.3
mpmath                       1.3.0
namex                        0.0.8
networkx                     3.4.1
numpy                        1.26.4
opencv-contrib-python        4.10.0.84
opencv-python                4.10.0.84
opt_einsum                   3.4.0
optree                       0.13.0
packaging                    24.1
pandas                       2.2.3
pillow                       10.2.0
pip                          24.0
proglog                      0.1.10
proto-plus                   1.26.1
protobuf                     5.29.4
pyasn1                       0.6.1
pyasn1_modules               0.4.2
PyAudio                      0.2.14
pydantic                     2.11.0
pydantic_core                2.33.0
Pygments                     2.18.0
pyparsing                    3.2.0
python-dateutil              2.9.0.post0
python-dotenv                1.1.0
pytz                         2024.2
requests                     2.32.3
rich                         13.9.2
rsa                          4.9
setuptools                   75.2.0
six                          1.16.0
SpeechRecognition            3.14.2
sqlparse                     0.5.1
sympy                        1.13.3
tensorboard                  2.17.1
tensorboard-data-server      0.7.2
tensorflow                   2.17.0
tensorflow-intel             2.17.0
termcolor                    2.5.0
torch                        2.2.2
torchvision                  0.17.2
tqdm                         4.66.5
typing_extensions            4.12.2
typing-inspection            0.4.0
tzdata                       2024.2
uritemplate                  4.1.1
urllib3                      2.2.3
Werkzeug                     3.0.4
wheel                        0.44.0
wrapt                        1.16.0

**Comment by Ayush-Sachan01** [comment]
Sure let me add these, it is important to maintain proper requirements.
